composer.json

```diff
@@ -152,7 +152,8 @@
         "symfony/security-acl": "~2.8|~3.0",
         "twig/cssinliner-extra": "^2.12|^3",
         "twig/inky-extra": "^2.12|^3",
-        "twig/markdown-extra": "^2.12|^3"
+        "twig/markdown-extra": "^2.12|^3",
+        "bjeavons/zxcvbn-php": "^1.0"
```

Comment:
Should be added in the validator component composer.json as well.

Reply:
Yes indeed I missed it. I have just added it as a dev dependency and in the `suggest` section as well.